src/silicone/multiple_infillers/decompose_collection_with_time_dep_ratio.py

```diff
@@ -139,18 +139,20 @@ def infill_components(
             database.
         """
         assert (
-            aggregate in to_infill_df.variables().values
+            aggregate in to_infill_df.variable
         ), "The database to infill does not have the aggregate variable"
         assert all(
-            y not in components for y in to_infill_df.variables().values
+            y not in components for y in to_infill_df.variable
         ), "The database to infill already has some component variables"
         assert len(to_infill_df.data.columns) == len(self._db.data.columns) and all(
             to_infill_df.data.columns == self._db.data.columns
         ), (
             "The database and to_infill_db fed into this have inconsistent columns, "
             "which will prevent adding the data together properly."
         )
-        self._filtered_db = self._db.filter(variable=components,)
+        self._filtered_db = self._db.filter(
+            variable=components,
```

Comment:
I think this is just a black quirk because of the pointless comma here, but I don't imagine it's worth our time fixing it.

Reply:
Yep, you can make a suggestion though and I can commit with one click to fix like the below

```suggestion
            variable=components
```

Reply:
Ok that didn't quite work (had to reformat to remove the new line), but now fixed locally